WordPress/UITestsFoundation/Screens/MySiteScreen.swift

```diff
@@ -84,7 +84,7 @@ public class MySiteScreen: ScreenObject {
         if XCUIDevice.isPad {
             removeButton = app.alerts.buttons.element(boundBy: 1)
         } else {
-            removeButton = app.sheets.buttons.element(boundBy: 0)
+            removeButton = app.buttons["Remove Site"]
```

Comment:
Out of scope for this PR, but it would be great if we could have accessibility identifiers for this and the "Remove" used above. cc @wordpress-mobile/mobile-ui-testing-squad